README.rst

```diff
@@ -1,15 +1,15 @@
 ####################
-pfioh - v1.5.14.2
+pfioh - v1.5.16.0
 ####################
 
 .. image:: https://badge.fury.io/py/pman.svg
-    :target: https://badge.fury.io/py/pman
+    :target: https://badge.fury.io/py/pfioh
```

Comment:
Not sure I understand this one

Reply:
So once upon a time I copy pasted from ``pman`` when I created the repo, and those references should have been pfioh for this repo. Does that make sense? Not sure what exactly you are asking about...

Reply:
Yes, but is line 5 still correct then?

Reply:
great catch!